resources/lib/uzg.py

```diff
@@ -216,5 +228,5 @@ def __rebuild_item(self, post, show_time_in_label):
 
             item = post
             item['label'] = titelnaam
-            item['video']['title'] = titelnaam
+            #item['video']['title'] = titelnaam
```

Comment:
ik denk dat je bij deze if: "if (item['video']['aired'] == ref['video']['aired'] and item['whatson_id'] != ref['whatson_id']):" wil controleren of je ['video']['title'] ook ongelijk is. Want als dat anders id hoeft de hele tijd notatie er niet in.
Dit doe ik zodat je b.v. het NOS JOURNAAL met de tijd er achter krijgt. anders zie je geen verschil tussen de uitzending van 16uur en 18uur.